### DISCUSSION TITLE
Has the "directus_presets.filters" column been renamed to "filter" in recent versions?

### DISCUSSION THREAD

**Discussion by powerseed** [directus/directus#9765] [Archive]
I saw this field renamed after updating to a newer version (from `9.0.0.rc-83` to `9.0.0.rc-99`), but I didn't find any info about it in release logs.

Was it renamed from "filters" to "filter" in any recent versions?

Thanks in advance!

**Comment by rijkvanzanten** [comment]
Yes, in migration 20211007A-update-presets
